contracts/interfaces/IVoucherKernel.sol

```diff
@@ -182,12 +176,6 @@ interface IVoucherKernel {
     function setVoucherSetTokenAddress(address _voucherSetTokenAddress)
         external;
 
-    /**
-     * @notice Set the general complain period, should be used sparingly as it has significant consequences. Here done simply for demo purposes.
-     * @param _complainPeriod   the new value for complain period (in number of seconds)
-     */
-    function setComplainPeriod(uint256 _complainPeriod) external;
-
```

Comment:
You should leave this in the interface. No problem if it's external here, but the actual implementation is public.

Reply:
Added back in.